chipy_org/apps/main/views.py

```diff
@@ -21,6 +21,9 @@ def get_context_data(self, **kwargs):
         future_meetings = Meeting.objects.filter(
             when__gt=datetime.datetime.now() - datetime.timedelta(hours=24))
 
+        context["general_sponsors"] = GeneralSponsor.objects.all(
+            ).order_by('?')
```

Comment:
ok, cool

On Sat, Feb 27, 2016 at 7:29 PM, Joe Jasinski <notifications@github.com>
wrote:

> In chipy_org/apps/main/views.py
> <https://github.com/chicagopython/chipy.org/pull/120#discussion_r54342570>
> :
>
> > @@ -21,6 +21,9 @@ def get_context_data(self, **kwargs):
> >          future_meetings = Meeting.objects.filter(
> >              when__gt=datetime.datetime.now() - datetime.timedelta(hours=24))
> >
> > +        context["general_sponsors"] = GeneralSponsor.objects.all(
> > +            ).order_by('?')
>
> That's Django's notation for random ordering. For small queries like this,
> I think this fits the bill (but for much larger queries, it might not be
> very efficient).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chicagopython/chipy.org/pull/120/files#r54342570>.
>



-- 
Brian Ray
@brianray
(773) 669-7717